src/common/feedGenerator.ts

```diff
@@ -151,12 +155,15 @@ export const feedToFilters = async (
       }
       return acc;
     },
-    { includeTags: [], blockedTags: [] },
+    { includeTags: [], blockedTags: [] } as {
+      includeTags: string[];
+      blockedTags: string[];
+    },
```

Comment:
Wouldn't this already work?
```suggestion
    { includeTags: string[], blockedTags: string[] }
```

Reply:
that would be syntax error, its not type its `acc` value of `reduce`, but I moved it to `reduce` generic above. 23504d826da8baaca0c014a0f891afb18d665bce